### DISCUSSION TITLE
Incorrect languages on my project

### DISCUSSION THREAD

**Discussion by Kayoshi-dev** [github-linguist/linguist#6937] [Classification] [answered]
Hello! 

The language displayed on my repo is incorrect (it's not only Javascript! It contains TS, Rust, Svelte...)

I think it happened right after I moved my project to an organization after I pushed a commit?

Anyway, here's the link :
https://github.com/FlawedLabs/inscribe

Thanks for your help!

**Comment by lildude** [comment, marked as answer]
> The language displayed on my repo is incorrect (it's not only Javascript! It contains TS, Rust, Svelte...)

It's not... it's showing that you have sooo much more JavaScript by bytes of code that all the other languages are grouped into the "Other 0.9%":

![Screenshot 2024-07-13 at 10 56 43](https://github.com/user-attachments/assets/4d4fc94c-45fe-406d-aae6-fd4608e678b3)

```console
➜ github-linguist inscribe/
99.15%  1402703    JavaScript
0.70%   9968       Svelte
0.06%   786        TypeScript
0.05%   739        Rust
0.03%   427        HTML
0.01%   113        CSS
➜ 
```

Most of this is due to one file - https://github.com/FlawedLabs/inscribe/blob/main/static/pdf.worker.min.mjs - which is 1.4MB.

You probably want to use an [override](https://github.com/github-linguist/linguist/blob/master/docs/overrides.md) to mark this as generated or vendored which will give you this:

```console
➜ github-linguist inscribe/
75.72%  9968       Svelte
8.60%   1132       JavaScript
5.97%   786        TypeScript
5.61%   739        Rust
3.24%   427        HTML
0.86%   113        CSS
➜ 
```

**Reply by Kayoshi-dev** [reply]
Oooooooh you're definitely right, I forgot about that JS file.

Thanks for the help, I didn't know I could mark a file as vendored.

Have a nice day!
